core/templates/dev/head/pages/practice_session/practice_session.html

```diff
@@ -4,6 +4,14 @@
   Practice Session: {{ topic_name }} - Oppia
```

Comment:
`topic_name` wasn't displayed in the title bar.

Reply:
It should be fixed now